filters/speech-tag/filter.py

```diff
@@ -0,0 +1,87 @@
+import operator
+import spacy
+
+from interfaces.SentenceOperation import SentenceOperation, SentenceAndTargetOperation
+from tasks.TaskTypes import TaskType
+from collections import defaultdict
+from typing import Union
+
+"""
+A filter on if the tokens contain specific speech tag a certain number of times.
+"""
+
+
+class SpeechTagFilter(SentenceOperation):
+    tasks = [TaskType.TEXT_CLASSIFICATION, TaskType.TEXT_TO_TEXT_GENERATION]
+    languages = ["en"]
+
+    def __init__(
+        self,
+        speech_tags: Union[list, str],
+        thresholds: Union[list, str],
+        operations: Union[list, str],
+        percentages: bool = False,
+    ):
```

Comment:
It might be helpful to add default arguments here. When all filters would be called together - it might not be feasible to provide separate arguments for each.

Reply:
What would you recommend for the defaults? I was considering adding some, but decided to only add one for the `percentages` flag since I couldn't think of a default case for the others.

Reply:
It does not have to be symbolic of any task - just to have placeholders. But I guess we could expect performance drops in longer sentences in general - so maybe verb & noun counts of more than 2 or 3 each?